src/main/java/org/takes/rs/RsWithType.java

```diff
@@ -40,6 +40,11 @@
 public final class RsWithType extends RsWrap {
 
     /**
+     * Type header.
+     */
+    private static final String HEADER = "Content-Type";
```

Comment:
@Bertram25 We're only using this constant once, let's leave it inlined as a literal `String`.